api-report/container-definitions.api.md

```diff
@@ -485,7 +485,7 @@ export interface IResolvedFluidCodeDetails extends IFluidCodeDetails {
 export interface IRuntime extends IDisposable {
     createSummary(blobRedirectTable?: Map<string, string>): ISummaryTree;
     getPendingLocalState(): unknown;
-    notifyAttaching(snapshot: ISnapshotTreeWithBlobContents): void;
+    notifyOpReplay(message: ISequencedDocumentMessage): Promise<void>;
```

Comment:
1. this is a breaking change, and a cross layer breaking change at that, so it will need to be staged
2. i want to spend some time thinking about the api as i continue through the review

Reply:
It may be technically breaking, but I don't think it actually breaks anything besides this feature between versions, which I believe is okay since it's currently not used. It shouldn't affect anyone who doesn't have `enableOfflineLoad` set to true.